pkg/storage/unified/search/bleve.go

```diff
@@ -532,9 +532,16 @@ func (b *bleveIndex) toBleveSearchRequest(ctx context.Context, req *resource.Res
 		facets[f.Field] = bleve.NewFacetRequest(f.Field, int(f.Limit))
 	}
 
+	// Sort fields need to be included as results in the hits
```

Comment:
Going to change this to make the caller add the sort field to the `field` url param if they want the sort field back in the results/hits.